app/src/nxt/java/com/pileproject/drive/execution/NxtController.java

```diff
@@ -194,31 +197,32 @@ public void move(MotorDir leftMotorDir, MotorDir rightMotorDir) {
     }
 
     /**
-     * set motor's power as percent
+     * A setter of motor power.
      *
      * @param kind
-     * @param percent
+     * @param power
      */
-    public void setMotorPower(MotorKind kind, double percent) {
-        if (percent > 100.0) {
-            percent = 100.0;
-        } else if (percent < 0) {
-            percent = 0.0;
+    public void setMotorPower(MotorKind kind, int power) {
+        if (power > 100) {
```

Comment:
I'm wondering whether this clipping is good for users of this class.
Which way do you think better, throwing exceptions or clipping values?
Anyway, at least we should note the way in doc.